### DISCUSSION TITLE
PushNotifications not returning the callback from the listener "registration"

### DISCUSSION THREAD

**Discussion by jdinartejesus** [capawesome-team/capacitor-plugins#146] [Q&A] [answered]
Hi everyone, I'm not able to get the callback from "registration" on the PushNotifications plugin, I'm using Vue and Capacitor V5. I' adding the listener to App.vue and Mounted() function, but no luck getting the Firebase token. Someone had this issue before or know where could be the issue?

**Comment by robingenz** [comment, marked as answer]
Which plugin do you mean exactly? There is no Push Notifications plugin in this repo.

**Reply by jdinartejesus** [reply]
I thought was possible to have from the Badge Counter? Or only returns the permissions?

**Reply by robingenz** [reply]
No, the Badge plugin does not support push notifications. You have to use the offical [Push Notifications](https://capacitorjs.com/docs/apis/push-notifications) plugin or the [Firebase Cloud Messaging](https://capawesome.io/plugins/firebase/cloud-messaging/) plugin. The Badge plugin only needs the permission to display the badges.

**Reply by jdinartejesus** [reply]
@robingenz got it, my bad! Thanks so much for such quick reply 🎉
